packages/core/src/css/create-themed-component.tsx

```diff
@@ -28,8 +28,8 @@ export function createThemedComponent<
     const {
       sx: _sx,
       as: SuperComponent,
-      variant,
       style,
+      variant = options.defaultVariant,
```

Comment:
sorry for the late review. this line can have unintended consequences i think. probably best to take it out i think

Reply:
That mean variant will be undefined by default ? I've added this to avoid that and to match with the variants props (L34) that have default value. So, I remove this ?